README.md

```diff
@@ -1,10 +1,10 @@
 # webpack-starter
 
-> **Demonstrates bundling [`level`](https://github.com/Level/level) for browsers using [`webpack`](https://webpack.js.org/).**  
+> **Demonstrates bundling [`level`](https://github.com/Level/level) for browsers using [`webpack`](https://webpack.js.org/).**
```

Comment:
Please keep this whitespace, it acts as a newline in markdown.

Reply:
Put that back in bd2db3b